### DISCUSSION TITLE
get_album() isn't working? doesn't even work with the test case

### DISCUSSION THREAD

**Discussion by adriangallant99** [sigma67/ytmusicapi#646] [Q&A]
`yt = YTMusic(settings.AUTH_FILEPATH)

search_results = yt.search('Oasis Wonderwall')

search_results = yt.get_album('MPREb_4pL8gzRtw1p')

print(search_results)`

I've tried debugging the API to the best of my ability and I can't get the get_album method to work. I mirrored your testing files as well as copy and pasted your code from the documentation. The search method works without issue. Get_album, however, doesn't work. I've even searched through your tests to use the same browser ID as the testers, but to no avail.

Please help!!!

**Comment by sigma67** [comment]
Can't debug without an error message
